vpnbook

```diff
@@ -39,30 +39,32 @@ __read_property(){
 }
 
 __zip_downloader(){
-	local base_name=VPNBook.com-OpenVPN
+	local base_name=vpnbook-openvpn
 	local country=$1 	
 	local ext=zip
-	local u_country=$(echo $country | tr '[:lower:]' '[:upper:]')
+	local u_country=$(echo $country | tr '[:upper:]' '[:lower:]')
 
 	if [ ! -f ${vpnbook_folder}/${u_country}/${base_name}-${country}.${ext} ]; then
 		echo "Downloading" ${base_name}-${country}.${ext}	    	
-		curl ${vpnbook_base_url}/free-openvpn-account/${base_name}-${country}.${ext} -o ${vpnbook_folder}/${base_name}-${country}.${ext} 
+		curl ${vpnbook_base_url}/free-openvpn-account/${base_name}-${u_country}.${ext} -o ${vpnbook_folder}/${base_name}-${u_country}.${ext} 
 		# Make the original variable uppercase					
-		unzip ${vpnbook_folder}/${base_name}-${country}.${ext} -d ${vpnbook_folder}/${u_country}	
-		mv ${vpnbook_folder}/${base_name}-${country}.${ext} ${vpnbook_folder}/${u_country}
-	fi
-}
+		unzip ${vpnbook_folder}/${base_name}-${u_country}.${ext} -d ${vpnbook_folder}/${u_country}	
+    mv ${vpnbook_folder}/${u_country}/vpnbook-openvpn-$u_country/* ${vpnbook_folder}/${u_country}
 
-__is_user_allowed_to_vpn(){
-	local group=$(id -G -n $USER | grep -o "vpnstarter")
-	if [ "${group}" == "vpnstarter" ]; then
-		echo "The user is part of the vpnstarter Group"
-	else
-		echo "The user is not part of the vpnstarter Group. Please Fix the issue first and then retry."
-		exit 1
+    mv ${vpnbook_folder}/${base_name}-${u_country}.${ext} ${vpnbook_folder}/${u_country}
 	fi
 }
 
+#__is_user_allowed_to_vpn(){
```

Comment:
I think this whole function might be removed

Reply:
Yup was planning to do so 
Kept it because i thought you would need it maybe?

Reply:
You can remove it. If I need it I will revert the change.

Reply:
Done